packages/util-error/lib/ValidationError.mjs

```diff
@@ -18,4 +18,13 @@ export class ValidationError extends UserFacedError {
   get message() {
     return this.#message
   }
+
+  toJSON() {
+    const original = super.toJSON()
+
+    return {
+      ...original,
+      errors: this.errors,
```

Comment:
why it is an errors and not just an error?

Reply:
Because it's a list, after running `validator` we can have a list of errors